### ISSUE TITLE
[Feature request] Add the last date the list was updated/changed and not only last time an update was launched

### ISSUE THREAD

**Issue by lordraiden** [AdguardTeam/AdGuardHome#4356]
### Problem Description
Right now when you have lot of lists is hard to know which of them are no longer being updated.

### Proposed Solution
 Add the last date the list was changed and not only last time an update was launched

**Comment by lordraiden** [comment]
This is easy to implement and is useful to detect dead feeds

**Comment by sjhgvr** [comment]
Agree. Currently all "Last time updated" times are the same for each list (time after the Check For Updates-button has pressed).
This makes no sense.

In my eye's it should be like this:

- Next to "Check for updates"-button a time should be displayed (This should be the time of the last update check, either by automated update, or manually pressing that button.)
- Next to each list, the actual time of last change to that particular list*

- Extract "TimeUpdated", "Last modified", "Last updated", "Last update", "Updated" - info, from the top of the downloaded file. (When available)
- HTTP Last-modified header? (Can be wrong)
- Use file-mod-time from the local file. If the downloaded file was changed, update the mod time. If not, leave the file alone. (Last time updated will not change)
